playwright-tests/tests/payments/proposal-details.spec.js

```diff
@@ -171,18 +176,43 @@ async function setupSandboxAndCreateProposal({ daoAccount, page }) {
   await sandbox.addPaymentRequestProposal({
     title: proposalTitle,
     summary: proposalSummary,
-    amount: "5",
+    amount: "4" + "000000000000000000000000",
```

Comment:
I think it is better then to write:

`4_0000_00000_00000_00000_00000n.toString()`